content/en/blog/_posts/2023-12-05-My-first-ever-KubeCon-KubeCon+CloudNativeCon-North-America-2023/index.md

```diff
@@ -0,0 +1,178 @@
+---
+layout: blog
+title: "My first-ever KubeCon - KubeCon+CloudNativeCon North America 2023"
+date: 2023-12-05
+slug: first-kubecon-cloudnativecon-virtual-experience
+---
+
+**Author:** Rajdeep Pal (Student, Techno India University)
+
+_This post is a guide for beginners to look at huge and overwhelming events like KubeCon._
+
+<img src="pic1.jpg" alt="KubeCon NA 2023 Poster">
+</br>
```

Comment:
Please use Markdown - https://www.digitalocean.com/community/tutorials/markdown-markdown-images has an example.

Reply:
![Alt text](https://raw.githubusercontent.com/myusername/myrepo/main/example.jpg)

Sir, will this syntax do? @sftim